omero/users/cli/installation.rst

```diff
@@ -4,16 +4,49 @@ Installation
 .. note:: The |CLI| is currently untested on Windows
     but may be supported in the future.
 
-We recommend installing in a Python virtual environment.
-Since OMERO 5.6, only Python 3 is supported::
+Since OMERO 5.6, only Python 3 is supported.
+We assume that you have already installed Python 3.6.
 
-    python3.6 -mvenv venv
-    . venv/bin/activate
+We recommend installing omero-py and the |CLI| plugins
+in a Python virtual environment.
+You can either create one using ``venv`` or ``conda`` (preferred).
+If you opt for Conda_, you will need
+to install it first, see `miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ for more details.
+
+To create a virtual env and install ``omero-py`` using venv::
+
+    python3.6 -m venv py3_venv
+    . py3_venv/bin/activate
     pip install omero-py
 
-    # omero command is now available
+To create a virtual env and install ``omero-py`` using conda (preferred)::
+
+    conda create -n myenv python=3.6
+    conda activate myenv
+    conda install -c ome zeroc-ice36-python 
+    conda install -c ome omero-py 
+
+The ``omero`` command is now available::
+
     omero login
 
-The |CLI| installed in this way provides all functionalities except import.
-Import requires jars in the OMERO.server package.
-See :doc:`/sysadmins/unix/server-installation`.
+The |CLI| installed in this way provides all functionalities except the ``import`` functionality.
+
+The ``import`` functionality requires to have a supported version of Java e.g. Java 11, Ice |iceversion| installed
```

Comment:
Replace with ``...requires to have a supported version of Java e.g. Java 11 and Ice (iceversion) installed. Also, it requires JARs....`` (better wording, at the moment, the Ice is a bit drowned in the sentence)